src/PeerPoolMgr.h

```diff
@@ -18,6 +18,20 @@ class HttpRequest;
 class CachePeer;
 class CommConnectCbParams;
 
+class PeerPoolMgrContext : public CodeContext
```

Comment:
Please see whether this class can be generalized into DetailedCodeContext class that takes an SBuf detail and a c-string gist as constructor parameters (and stores that info in its data members, to be reported using CodeContext APIs). If that works, use DetailedCodeContext instead of CachePeerProbeCodeContext as well.

Eventually, CachePeer will be reference-counted and, hence, may not need a dedicated CodeContext data member, but (re)using DetailedCodeContext for CachePeer now may help us make progress while those (complex) changes are pending (and waiting other/stuck smooth reconfiguration PRs).

Reply:
Done.